CoreScriptsRoot/CoreScripts/PurchasePromptScript2.lua

```diff
@@ -252,7 +278,8 @@ local function createImageButtonWithText(name, position, image, imageDown, text,
 	imageButton.ZIndex = 8
 	imageButton.Modal = true
 
-	local textLabel = createTextLabel(name.."Text", UDim2.new(1, 0, 1, 0), UDim2.new(0, 0, 0, 0), font, isTenFootInterface and largeFont or Enum.FontSize.Size24, text)
+	fontSize = fontSize or (isTenFootInterface and largeFont or Enum.FontSize.Size24)
```

Comment:
this is starting to get a little crazy to read, I think we should maybe put some ifs in here